drivers/bridge/bridge.go

```diff
@@ -218,14 +225,24 @@ func (c *networkConfiguration) fromLabels(labels map[string]string) error {
 			if c.EnableICC, err = strconv.ParseBool(value); err != nil {
 				return parseErr(label, value, err.Error())
 			}
-		case DefaultBridge:
-			if c.DefaultBridge, err = strconv.ParseBool(value); err != nil {
-				return parseErr(label, value, err.Error())
-			}
 		case DefaultBindingIP:
 			if c.DefaultBindingIP = net.ParseIP(value); c.DefaultBindingIP == nil {
 				return parseErr(label, value, "nil ip")
 			}
+		case DefaultBridgeLegacyIPv6Network:
```

Comment:
That would be the ideal solution. But we do not have the ipam options for Endpoints.
Second option is that libnetwork does it, but I'd refrain for adding bridge driver specific code into libnetwork.